sockeye/inference.py

```diff
@@ -1363,6 +1369,9 @@ def _beam_search(self,
         lengths = mx.nd.ones((self.batch_size * self.beam_size, 1), ctx=self.context)
         finished = mx.nd.zeros((self.batch_size * self.beam_size,), ctx=self.context, dtype='int32')
 
+        # Extending max_output_lengths to shape (batch_size * beam_size,)
+        max_output_lengths = mx.nd.repeat(max_output_lengths, self.beam_size)
```

Comment:
what's the relation to ```max_output_length``` (w/o trailing s) now? 
would it be correct to set ```max_output_length = nd.max(max_output_lengths)``` before the repeat?

Reply:
yes, that's the case. Maybe we should indeed make this explicit the way you wrote it.